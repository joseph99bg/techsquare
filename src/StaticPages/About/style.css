.about-page {
    max-width: 80%;
    margin: 30px auto 100px auto;
}
.about-page h1 {
    font-family: 'Orbitron';
    color: #20aaea;
    font-weight: 600;
    font-size: 40px;
}
.about-page p {
    font-size: 18px;
    font-family: 'Montserrat';
    font-weight: 400;
}
.about-page li {
    font-size: 18px;
    margin: 5px 0;
    font-family: 'Montserrat';
}