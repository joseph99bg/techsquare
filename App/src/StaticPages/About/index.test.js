import React from 'react'
import About from './index'
import renderer from 'react-test-renderer'

describe('About component', () => {
    test('should render correctly', () => {
        const component = renderer.create(
            <About />
        )
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})