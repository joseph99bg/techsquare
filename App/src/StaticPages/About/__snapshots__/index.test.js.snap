// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should render correctly 1`] = `
<div
  className="about-page"
>
  <h1>
    About TechSquare
  </h1>
  <p>
    TechSquare is a blog about all recent news and trends around technology, gadgets and programming.
  </p>
  <p>
    It is open for public use and anyone can post in it. But there are some requirements:
  </p>
  <ol>
    <li>
      You should only post relevant to the site theme news and trends.
    </li>
    <li>
      You should not be violent to other users.
    </li>
    <li>
      You should write in English.
    </li>
    <li>
      You should be friendly :)
    </li>
    <li>
      It is good if you add and image to your post (optional)
    </li>
  </ol>
  <p>
    Thank you for your attention. Happy blogging!
  </p>
</div>
`;
